README.md

```diff
@@ -138,6 +138,24 @@ The context added looks like:
 
 **Note:** `Cloud role name` and `Cloud role instance` shown in the screenshot above, represent service and service instance respectively, and are special fields in Application Insights schema that are used as aggregation units to power certain experiences like Application map.  
 
+### Event Correlation with Service Remoting
+You will have enhanced experience with correlated events across services. If you're using HttpClient to communicate between your services, the correlation is supported implicitly. For Service Remoting, you just need to make some minor changes to your existing code and the correlation will be supported.

```

Comment:
The first sentence is a bit unclear--it is not obvious what the "enhanced experience" is. Consider something like this:

The Nuget package enables correlation of traces produced by Service Fabric services, regardless whether services communicate via HTTP or via Service Remoting protocol.